modules/improvedigitalBidAdapter.js

```diff
@@ -307,10 +307,10 @@ const ID_REQUEST = {
     }
     // In the single request mode, split imps between those going to the ad server and those going to extend server
     if (extendImps.length) {
-      requests.push(formatRequest(extendImps, null, true));
+      requests.push(formatRequest(extendImps, bidderRequest.auctionId, true));
```

Comment:
https://github.com/prebid/Prebid.js/blob/f407e47f96f12834bd79092bc1b5bd0203967598/modules/improvedigitalBidAdapter.js#L165 seems to still be there @dgirardi

Reply:
@patmmccann, I missed that, but I don't think it has any effect now, because it's always overridden by this PR setting it to auctionID afterwards. Since all of that is being refactored in https://github.com/prebid/Prebid.js/pull/8738 I think it's OK to leave.

Reply:
no problem; I changed the original issue to call out a bunch that i found where source was on one line and tid was on the next line